backend/src/main/resources/graphql/main.graphqls

```diff
@@ -279,6 +284,8 @@ type Query {
   ): Int @requiredPermissions(allOf: ["READ_RESULT_LIST"])
   testResult(id: ID!): TestResult
     @requiredPermissions(allOf: ["READ_RESULT_LIST"])
+  dashboardMetrics(facilityId: ID, startDate: DateTime!, endDate: DateTime!): TestMetrics
```

Comment:
I might call these something more specific like topLevelDashboardMetrics? Because I think we might need additional graphQL queries for other parts of the dashboard, like the facility breakdown table.